src/components/ChatEntry.js

```diff
@@ -1,22 +1,53 @@
-import React from 'react';
+import { React, useState } from 'react';
 import './ChatEntry.css';
 import PropTypes from 'prop-types';
+import TimeStamp from './TimeStamp';
+
+const ChatEntry = ({ sender, body, liked, timeStamp, id, onUpdateMessage }) => {
+  const [isLiked, setIsLiked] = useState(liked);
+
+  if (isLiked) {
+    console.log('is liked');
+  } else {
+    console.log('is unliked');
+  }
+
+  const updateLiked = () => {
+    console.log('updating heart')
+    setIsLiked(!isLiked);
+    onUpdateMessage({
+      sender: sender,
+      body: body,
+      timeStamp: timeStamp,
+      id: id,
```

Comment:
hmm you are giving this function a lot of editing power when it only needs to update the `isLiked` property. Think about giving the function access only to `isLiked`.